### ISSUE TITLE
Repository Tab still saying the repo does not exists

### ISSUE THREAD

**Issue by thom4parisot** [odd-poet/redmine-gitolite#2]
Hello,

I've cloned the latest redmine-gitolite on a 1.1.1 redmine install.
I think I achieved most of the parts as :
- I created a user and added SSH keys
- I created a project, then a Git repository from redmine
- I followed instructions on the repository tab (add user to the project etc.)
- I've pushed successfully data to the new project

However, the repository tab still says « Repository does not exist. Create one using the instructions below. »

I've checked in the git git/repositories dir: the myproject.git folder is here.
When I launch a `rake gitolite:update_repositories` task, I see "Updating repositories for projects MyProject". However, when I launch `rake gitolite:fetch_changes` it tells me /path/to/git/repositories/myproject.git/ is not a git repository.

I guess I'm missing one point, or maybe it's a but?

Thanks :-)

**Comment by odd-poet** [comment]
Maybe redmine could not access git repositories. 

could you add group 'redmine' user to 'git' group? 
e.g. sudo addgroup redmine git 

Thank.

**Comment by thom4parisot** [comment]
Thanks for your answer.

Unfortunately it's already the case.

**Comment by odd-poet** [comment]
1. Did you configure git repositories path? 
   (in redmine admin > plugins > configure) 
2. is there no permission error?
   'sudo -H -u redmine ls /path/to/git/repositories/your_project.git'

**Comment by thom4parisot** [comment]
Hello :) 

the path is fine. I checked the permissions and indeed, all perms of the git repos are set for the git _user_, and nothing for the git _group_ (700 for folders, 600 for files).

As I added the same perms to git group, it's fine.
But I'm wondering if it's a normal manipulation to do.

I have a git user (used by gitolite), a redmine user (ran through passenger) and a www-data (which runs the webserver).
